pkg/crc/machine/bundle/metadata.go

```diff
@@ -353,7 +353,12 @@ func Download(preset crcPreset.Preset, bundleURI string) (string, error) {
 	// bundles, their sha256sums are known and can be checked.
 	if bundleURI == constants.GetDefaultBundlePath(preset) {
 		if preset == crcPreset.OpenShift || preset == crcPreset.Microshift {
-			return DownloadDefault(preset)
+			downloadedBundlePath, err := DownloadDefault(preset)
```

Comment:
Is there a way to differentiate "not found" from other errors?

Reply:
@cfergeau looks like `grab` package have `StatusCodeError` type and can validated with `IsStatusCodeError` which I now used.
```
StatusCodeError indicates that the server response had a status code that
// was not in the 200-299 range (after following any redirects).
```